templates/coderd.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
     app.kubernetes.io/component: {{ include "coder.serviceName" . }}
     app: {{ include "coder.serviceName" . }}
     coder.deployment: {{ include "coder.serviceName" . }}
-  annotations: {{ toYaml .Values.services.annotations | nindent 4 }}
+  annotations: {{ toYaml .Values.coderd.annotations | nindent 4 }}
```

Comment:
I think we can use the merge function for this, I haven't tried it but we might be able to do something like:

`{{ toYaml (merge .Values.services.annotations .Values.coderd.annotations) | nindent 4 }}`

https://helm.sh/docs/chart_template_guide/function_list/#merge-mustmerge